src/ifp/src/InitFloorplan.tcl

```diff
@@ -352,16 +357,16 @@ proc make_die_helper { key_array } {
 proc make_polygon_die_helper { key_array } {
   array set keys $key_array
 
-  if { ![info exists keys(-die_polygon)] } {
+  if { ![info exists keys(-die_area)] } {
     utl::error IFP 75 "no -die_polygon specified for polygon floorplan."
```

Comment:
The -die_polygon option doesn't exists anymore. Please update the error message.